src/include/kdbprivate.h

```diff
@@ -15,6 +15,7 @@
 #include <kdbhelper.h>
 #include <kdbio.h>
 #include <kdbmacros.h>
+#include <kdbnotificationinternal.h>
```

Comment:
Do we need the internal header file only for a callback?

Reply:
Since the kdbnotification.h is installed and user-facing I didn't want to include these definitions there. They are only used internally by the internalnotification plugin, the notification library and transport plugins.